app/javascript/shared/components/Branding.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     v-if="globalConfig.brandName && !disableBranding"
-    class="px-0 py-3 flex justify-center"
+    class="px-0 py-3 flex justify-center bg-slate-25 dark:bg-slate-800"
```

Comment:
This is a shared component, any edits here should be made in the parent component.

This change breaks the survey component
<img width="630" alt="Screenshot 2023-09-08 at 1 40 50 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/eb5d8a71-199a-4513-914f-93d2a437f3df">